### ISSUE TITLE
Let current values be default when editing items.

### ISSUE THREAD

**Issue by drasbeck** [aesophor/py-todo#19]
As of 1.3 the current values (Title, Date) are not used as default, when editing items. This means that one has to rewrite the name, to edit the deadline and vice versa. This is trivial for short items, but cumbersome for longer titled items.

One solution could be to let current values be the default, and show them in parentheses when prompting, so that pressing enter when prompted for either, that value is kept.

An editing scenario with index 1 item named "Linear Algebra Test, Pages 12-666, Bldg 13 Room 101" would look like:
```
$ todo -e 1
Title (Linear Algebra Test, Pages 12-666, Bldg 13 Room 101): |
```
Instead of the current:
```
$ todo -e 1
Title: |
```
So that simply pressing enter keeps the current title, whereas it currently changes the item to have an empty title.

**Comment by aesophor** [comment]
Thanks! Solved it in commit 86d47c8b36ead2b397b4c26e89b8415020d50eb2!

**Comment by aesophor** [comment]
> Lol I actually had this idea and I have it in an uncommitted PR. It'll be in my next one.

You guys are really creative! This program really has improved a lot with your help :tada:
